### PR TITLE
babel-preset-es2015 to babel-preset-env

### PR DESCRIPTION
**Pull request by alirezavalizade**

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit d9190316962cb26a77c273ae76e89ea246e6fdc1

https://deploy-preview-85--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please add a [changelog.md](https://github.com/pinterest/gestalt/blob/master/CHANGELOG.md) entry for your changes.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by chrislloyd** [comment]
@alirezavalizade 🙏 thank you very much!

@christianvuerings does this affect the `.broserlistrc` change that you have planned?

File: package.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
     "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-plugin-webpack-loaders": "0.9.0",
     "babel-polyfill": "^6.23.0",
-    "babel-preset-es2015-loose": "^8.0.0",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-env": "^1.6.1",
```
Comment (christianvuerings) [inline package.json]: @alirezavalizade Thank you for submitting this PR!

Updating `package.json` should have resulted in an updated `yarn.lock` file.

Did you run the following?
```
yarn remove babel-preset-es2015-loose babel-preset-es2015
yarn add --dev babel-preset-env
```

Comment (alirezavalizade) [reply package.json]: No i'm not modified yarn.lock, I will fix that sooner.

**Pull request merged**